components/Repositories/RossChart.tsx

```diff
@@ -102,7 +109,7 @@ function CustomTooltip({ active, payload }: TooltipProps<ValueType, NameType>) {
           <section className="flex justify-between">
             <p className="flex gap-2 items-center px-1 text-slate-500">
               <span className={`w-2 h-2 rounded-full bg-[#60A5FA] inline-block`}></span>
-              New:
+              Outside:
```

Comment:
This copy change is 👍🏼 

The API probably needs an update since we end up calling the select for "internal contributors" to "new"

```ts
      .addSelect("outside_contributors.contributors", "new")
      .addSelect("returning_contributors.contributors", "recurring")
      .addSelect("org_contributors.contributors", "internal")f
```

where we look at the pr author association. Just making this note for myself since it'd be a good idea to make the API more clear in what it's doing for the ROSS